modules/material-management/test/cryptographic_material.test.ts

```diff
@@ -154,16 +123,12 @@ describe('decorateCryptographicMaterial', () => {
     const suite = new NodeAlgorithmSuite(
       AlgorithmSuiteIdentifier.ALG_AES128_GCM_IV12_TAG16
     )
-    const test = decorateCryptographicMaterial(
-      { suite, keyringTrace: [] } as any,
-      KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY
-    )
+    const test = decorateCryptographicMaterial({ suite } as any)
     test.zeroUnencryptedDataKey()
     const dataKey = new Uint8Array(suite.keyLengthBytes).fill(1)
     const trace = {
       keyNamespace: 'k',
       keyName: 'k',
-      flags: KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY,
     }
     // It is very hard to test this perfectly.  However, this tests the spirit.
     expect(() =>
```

Comment:
Same as above: shouldn't `trace` be deleted entirely? (sorry the indicated diff is in the wrong place, GitHub won't let me add a comment on the correct line)